### ISSUE TITLE
Timing of the talks

### ISSUE THREAD

**Issue by D-32** [swissmobidevs/appbuilders16#18]
Some talks went on for too long, we should maybe setup a monitor so that speakers see how much time they have left. This would also help to have both room more synchronised. We had some cases where the talk upstairs finished and downstairs had already started or was still finishing the previous one. Wasn't clear for attendees if they should wait outside or already go in.

**Comment by akosma** [comment]
We definitely have to monitor that. I'm the first guilty in here, my talk went on for almost 40 min. Sorry about that.

When one is speaking is hard to tell time – having a big iPad Pro with a timer on it would definitely help speakers. Or at least me :) lol

Related note: as you can see, we need plenty of iPad Pros for next year; for checking in, for timing, for schedules… boom.

**Comment by bontoJR** [comment]
@akosma yeah, a lot of iPad Pros that you can definitely kindly gift to us, considering you're a rockstar now! :trollface:
